samples/sample_hevc_fei/src/sample_hevc_fei.cpp

```diff
@@ -596,13 +650,20 @@ mfxStatus CheckOptions(const sInputParams& params, const msdk_char* appName)
         PrintHelp(appName, "Unsupported NumMvPredictorsL1 value (must be in range [0,4])");
         return MFX_ERR_UNSUPPORTED;
     }
-    if (!(params.encodeCtrl.NumFramePartitions == 1 ||
-          params.encodeCtrl.NumFramePartitions == 2 ||
-          params.encodeCtrl.NumFramePartitions == 4 ||
-          params.encodeCtrl.NumFramePartitions == 8 ||
-          params.encodeCtrl.NumFramePartitions == 16  ))
+    std::list<mfxU16> nfp_values = {1, 2, 4, 8};
```

Comment:
well, you actually don't need any stl container, just array would be fine. Since we don't have set::contains (which is sad), I believe best and simplest way would be:
```
mfxU16 nfp_values[] = {1,2,4,8}; // you could benefit from the fact that they are sorted
if (std::binary_search(std::begin(nfp_values), std::end(nfp_values), params.frameCtrl.CtrlI.NumFramePartitions) == std::end(nfp_values))
```

But I think now code is good enough to be merged